### ISSUE TITLE
Rework how parquet export is orchestrated.

### ISSUE THREAD

**Issue by matsonj** [matsonj/nba-monte-carlo#32]
Today there is a pretty janky parquet integration. What I think is a better way to do this is something like the following:

1. use ```target-parquet``` in meltano, and remove ```target-duckdb```. This allows a cleaner interface into duckdb, and also hedges against stability problems with the duckdb database format.
2. add a dbt tag (?) to identify models to export.
3. add a var to ```dbt_project.yml``` for ```export_to_parquet: true``` (or false) if you don't want to export.
4. run the entire project inside of duckdb.
5. run a macro to export files, either as a ```run-operation``` or via ```on-run-end```. The macro would grab a list of all models with a certain tag, then loop through exporting them one at a time.

This allows for a much cleaner set of dbt models. It also is a clear hand-off into some other system to handle the parquet files. As an example, you could orchestrate this pretty cleanly with meltano invoking the ```run-operation``` after ```dbt build``` completes.

**Comment by matsonj** [comment]
The question is - to use the duckdb database format or just run it in-memory? I lean towards in-memory, but that adds the trade-off that a ```run-operation``` will not be possible because the process will be closed.

**Comment by matsonj** [comment]
As it were, no 'easy' way to get tags into a python list. So instead I will use [get_relations_by_pattern](https://github.com/dbt-labs/dbt-utils#get_relations_by_pattern-source) and tag tables that I want to export to the 'export' schema.

**Comment by matsonj** [comment]
Success! 
![image](https://user-images.githubusercontent.com/16811433/199391199-04c43b51-3372-4af3-88b7-27abc8be8121.png)

**Comment by matsonj** [comment]
Previously, parquet was a parallel project running inside dbt. It was implemented via a target, i.e. ```dbt run --target parquet```. This would trigger all of the parquet functionality.

First, the ```elo_rollforward``` procedure had logic to handle it one way if using parquet and another for duckdb. There is now only one path.

This pattern repeats. How models were materialized, how they related to each other (using ```depends_on```), and hardcoded references to file locations for the parquet files for interim models have been completely removed. 

One thing I kept was "ephemeral" as primary materialization. Since this project uses duckdb's ```:memory:``` database functionality,  there are only two materializations that matter, ephemeral and tables. We have to keep the tables so that the data can be exported later to parquet. This is roughly the same speed as using the parquet target.
